specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.generatedetailedcostreport.json

```diff
@@ -286,6 +286,14 @@
           "description": "The status of the long running operation.",
           "$ref": "#/definitions/Status"
         },
+        "startTime": {
```

Comment:
There are a bunch of breaking changes reported on this PR, as a result, asking you to make these changes in NEW API Version. Please get a breaking changes approval from Azure Breaking Changes Reviewers <azbreakchangereview@microsoft.com> and ARM can review the changes afterwards. Once you have the approval, please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers.

Reply:
@visingla-ms We already have the startTime and endTime in response. It was just not in the documentation and hence flagged.

[OpenAPI Hub - Correctness Issue: OBJECT_ADDITIONAL_PROPERTIES identified in operation GenerateDetailedCostReportOperationStatus_Get of MICROSOFT.COSTMANAGEMENT on API version 2022-10-01 (azure-devex-tools.com)](https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=EDADC2EB-B9AD-4207-B62B-F6995785CEE6)